### DISCUSSION TITLE
How can I change the Artist name?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [cmus/cmus#1322] [Troubleshooting]
![image](https://github.com/cmus/cmus/assets/114720383/d90a259d-1fa1-44f9-a5fe-e3444fced65b)
I want to change the <No Name>.

**Comment by MikeSS8** [comment]
File tagging seems to be outside CMUS' purview.  For myself, it's enough to do something like this:

`cmus-remote -Q | rg '^file ' | cut -f2- -d' ' | xargs tageditor`

**Comment by seamusdemora** [comment]
I use [`mp3tag`](https://www.mp3tag.de/en/) - it's brilliant!
